paths/electron/distributing/outline.md

```diff
@@ -1,3 +1,18 @@
-## Learning goals
+# Electron 103: Sharing Apps
 
-How, when, and where to share existing packaged apps legally and fruitfully
+So I have a packaged app, now what? How to get apps onto the app store and easy to download.
+
+## Native App Discovery
+- Apple has app store
+- Windows has Windows Store (https://www.microsoft.com/en-US/store/apps?rtc=1)
+- Linux has Ubuntu Apps Directory (https://apps.ubuntu.com/cat/) and Deepin App Store (http://appstore.deepin.com/)
+
+## Legal things to be aware of
+- Once you're sharing your app, your licenses and other legal things need to be in order
+   - App submission checklist for Windows (https://docs.microsoft.com/en-us/windows/uwp/publish/app-submissions) 
+
+## Downloads from a website
+
+### How to set up a website so others can download a program by clicking a button
```

Comment:
@brianamarie thoughts on moving the creating your own site before getting on an app store? just an idea. getting your app approved by any of the stores isn't a 15 minute experience, so having them create the website is something they can actually publish by the end of the course.